### PR TITLE
Add non-DXO shareable support to FLModelUtil

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description 

- Support raw Shareable for FLModelUtils

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/apis/dxo.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ class DataKind(object):
     COLLECTION = "COLLECTION"  # Dict or List of DXO objects
     STATISTICS = "STATISTICS"
     PSI = "PSI"
+    META = "META"  # only contains meta
```
Comment (yanchengnv) [inline nvflare/apis/dxo.py:33]: In this case, I'd think it should be "NONE" meaning that there is no data.

File: nvflare/app_common/ccwf/cse_client_ctl.py [hunk 2]
```diff
@@ -216,8 +217,10 @@ def _do_process_get_model_request(self, request: Shareable, fl_ctx: FLContext) -
             return make_reply(ReturnCode.BAD_REQUEST_DATA)
 
         if not self.local_model:
+            dxo = DXO(data_kind=DataKind.META)
```
Comment (yanchengnv) [inline nvflare/app_common/ccwf/cse_client_ctl.py:220]: I don't see why this is needed.

**Review by yanchengnv — CHANGES_REQUESTED** [review]
I don't understand what this PR is for.

File: nvflare/app_common/utils/fl_model_utils.py [hunk 3]
```diff
@@ -111,7 +111,7 @@ def from_shareable(
```
Comment (YuanTingHsieh) [inline nvflare/app_common/utils/fl_model_utils.py:111]: fix it here, need to accept "RAW" Shareable, but not just DXO

**Comment by yanchengnv** [comment]
I don't think we need this PR.

**Comment by YuanTingHsieh** [comment]
As discussed, we should only pass the shareable header in the case of "cross site evaluation"

**Pull request closed without merging**